planner/core/task.go

```diff
@@ -872,7 +872,12 @@ func (p *PhysicalSort) attach2Task(tasks ...task) task {
 }
 
 func (p *NominalSort) attach2Task(tasks ...task) task {
-	return tasks[0]
+	if p.OnlyColumn {
+		return tasks[0]
+	}
+	t := tasks[0].copy()
+	t = attachPlan2Task(p, t)
```

Comment:
Could we call `TurnNominalSortIntoProj` here?

Reply:
I don't know the impact of doing this. But I think `TurnNominalSortIntoProj` to `NominalSort` is the same as `InjectProjBelowSort` to `PhysicalSort`. Could you tell me the reason?